parlai/agents/transformer/transformer.py

```diff
@@ -61,20 +65,44 @@ def add_cmdline_args(cls, argparser):
                                 'when using transformer to encode memories')
         # model specific arguments
         agent.add_argument('--normalize-sent-emb', type='bool', default=False)
-        agent.add_argument('--share-encoders', type='bool', default=False)
+        agent.add_argument('--share-encoders', type='bool', default=True)
         agent.add_argument('--has-memories', type='bool', default=False,
                            help='If true, text contains newline separated memories '
                                 'before the actual text')
         agent.add_argument('--use-memories', type='bool', default=False,
                            help='If true, use the memories to help with predictions')
         agent.add_argument('--scores-norm', choices={'dot', 'sqrt', 'dim'},
```

Comment:
Let's drop this option while we're at it. Braden and I found dot was best.

Reply:
Interestingly --embeddings-scale we seemed to get mixed results on though. I've consistently found it is good.